packages/core-data/src/resolvers.js

```diff
@@ -517,7 +517,12 @@ export const canUser =
 			resourcePath =
 				entityConfig.baseURL + ( resource.id ? '/' + resource.id : '' );
 		} else {
-			resourcePath = `/wp/v2/${ resource }` + ( id ? '/' + id : '' );
+			// Allows checking permissions for non-default (`/wp/v2`) namespaces
+			// such as `/wp-block-editor/v1`.
+			const isCustomNamespace = !! resource?.startsWith( '/' );
+			resourcePath =
+				( isCustomNamespace ? resource : `/wp/v2/${ resource }` ) +
+				( id ? '/' + id : '' );
```

Comment:
This was resolved mainly by entity resource object support, but a couple of REST APIs that use custom namespaces remain in the core. While not ideal, using an entity registry for the export endpoint doesn't make sense.

@youknowriad, @tyxla, @jsnajdr I would love to hear your thoughts on this. Should I extract it into a separate PR?

Reply:
I might not understand correctly, but similar to [`getNAvigationFallbackId`](https://github.com/WordPress/gutenberg/blob/af11c59da78a52bf4203733bdeabdbf5dad4315e/packages/core-data/src/resolvers.js#L756-L760), what do you think of the approach of creating something like `canExport` as a private API and accessing the v1 endpoints through that?

Reply:
That's one option. Another is to use the `rel` link in the theme endpoint - https://core.trac.wordpress.org/ticket/57379#comment:10.